app/src/main/java/at/roteskreuz/stopcorona/model/entities/configuration/DbConfiguration.kt

```diff
@@ -30,7 +30,16 @@ data class DbConfiguration(
     /**
      * Time of updating.
      */
-    val cacheTime: ZonedDateTime = ZonedDateTime.now()
+    val cacheTime: ZonedDateTime = ZonedDateTime.now(),
+
+    val exposureConfigurationMinimumRiskScore: Int = 1,                                                    // 1,
+    val exposureConfigurationDailyRiskThreshold: Int = 30,                                                 // 30,
+    val exposureConfigurationAttenuationDurationThresholds: List<Int> = listOf(33,53),                     // [33, 63],
```

Comment:
Didn't you want to use nullable instead?

Reply:
Merge https://github.com/austrianredcross/stopp-corona-android/pull/124 and check again.

Reply:
@falkorichter